src/locale/locales/tr/messages.po

```diff
@@ -2517,7 +2517,7 @@ msgstr "Karart"
 
 #: src/view/screens/AccessibilitySettings.tsx:111
 #~ msgid "Disable autoplay for GIFs"
-#~ msgstr ""
+#~ msgstr "GIF'lerin otomatik oynatılmasını devre dışı bırak"
```

Comment:
It's `GIFlerin` at line `1143`, and `GIF'lerin` here, I'm not sure about the correct version but I think it would be nice to keep it consistent